planb/cli.py

```diff
@@ -78,8 +82,45 @@ def create(regions: list,
     create_cluster(options=locals())
 
 
-sns_topic_help = 'SNS topic name to send Auto-Recovery notifications to'
-sns_email_help = 'Email address to subscribe to Auto-Recovery SNS topic'
+@cli.command()
+@click.option('--region', type=str, required=True)
+@click.option('--cluster-name', type=str, required=True)
+@click.option('--cluster-size', default=3, type=int, help='number of nodes per region, default: 3')
```

Comment:
I think extend command could work with dc_suffix or without. Should'nt we name this parameter differently, e.g. `node_count`?